src/js/stores/DCOSStore.js

```diff
@@ -97,6 +106,37 @@ class DCOSStore extends EventEmitter {
     MarathonStore.fetchServiceVersions(serviceID);
   }
 
+  onMarathonDeploymentsChange() {
+    if (!this.data.dataProcessed) {
+      return;
+    }
+    let deploymentsList = MarathonStore.get('deployments');
+    let serviceTree = MarathonStore.get('groups');
+
+    NotificationStore.addNotification(
+      'services-deployments',
+      'deployment-count',
+      deploymentsList.getItems().length
+    );
+
+    // Populate deployments with affected services
+    this.data.marathon.deploymentsList = deploymentsList
+      .mapItems(function (deployment) {
+        let ids = deployment.getAffectedServiceIds();
+        let services = ids.reduce(function (memo, id) {
+          let service = serviceTree.findItemById(id);
+          if (service != null) {
+            memo.push(service);
+          }
+          return memo;
+        }, []);
+
+        return Object.assign({affectedServices: services}, deployment);
+      });
+
+    this.emit(DCOS_CHANGE);
```

Comment:
Would it be better to have a different event name? Otherwise anything that listens for `DCOS_CHANGE`.
We typically have a new event name per endpoint and it looks like the deployments data comes from a different endpoint than the list of running apps.